crates/hive-utils/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# TODO: Use official madara image when https://github.com/keep-starknet-strange/madara/issues/982 is closed
+# FROM ghcr.io/keep-starknet-strange/madara:v0.1.0.experimental.3 as madara
+FROM fredtupas/madara as madara
+
+FROM ghcr.io/kkrt-labs/kakarot/deployer:latest as deployer
+
+FROM ghcr.io/kkrt-labs/kakarot-rpc/node:latest
+
+# Install svn to download Madara genesis config files
+RUN apt-get -y update; \
+    apt-get install -y --no-install-recommends subversion; \
+    apt-get autoremove -y; \
+    apt-get clean; \
+    rm -rf /var/lib/apt/lists/*
+
+# Copy Madara binary
+COPY --from=madara /madara-bin /madara-bin
+
+# Madara Path
+ENV MADARA_PATH=/root/.madara
+ENV MADARA_CONFIG_PATH=${MADARA_PATH}/configs
+ENV KAKAROT_CONTRACTS_PATH=${MADARA_PATH}/cairo-contracts/kakarot
+
+# Download Madara genesis config to source path
+RUN svn export https://github.com/keep-starknet-strange/madara/trunk/configs ${MADARA_CONFIG_PATH}
+
+# Copy combined genesis.json to source path
+COPY .hive/genesis.json ${MADARA_CONFIG_PATH}/genesis-assets/genesis.json
+
+# Copy compiled Kakarot contracts to destination path of compiled contracts
+COPY --from=deployer /app/kakarot/build ${KAKAROT_CONTRACTS_PATH}
+
+# RPC environment variables
+ENV KAKAROT_HTTP_RPC_ADDRESS="0.0.0.0:3030"
```

Comment:
is this fixed?

Reply:
Let me check what port Hive uses for the testing actually

Reply:
So for hive tests, it uses [8545](https://github.com/ethereum/hive/blob/master/simulators/ethereum/rpc/helper.go#L50) also is consistent with one of the [clients](https://github.com/ethereum/hive/blob/706909584a76aa0609f171e74108ac47497bcd04/clients/nethermind/mkconfig.jq#L72-L98)